src/components/Panel/Panel.ts

```diff
@@ -55,10 +55,13 @@ namespace fabric {
       this._panelHost._overlay.overlayElement.addEventListener("click", () => {
         this.dismiss();
       });
-
       let closeButton = this._panel.querySelector(".ms-PanelAction-close");
+
       if (closeButton !== null) {
-        closeButton.addEventListener("click", () => {
+        let closeButtonClone = closeButton.cloneNode(true);
```

Comment:
Maybe we should just remove the listener if it's there? Does this clone the element several times?